<?xml version="1.0"?>
<launch>

    <arg name="uavId" default="0"/>
    <arg name="altitude" default="5.0"/>
    <arg name="taskTypes" default="0,1,2,3,4"/>
    <arg name="taskAssignAtSeconds" default="5"/>
    <arg name="serialPort" default="/dev/ttyUSB0"/>
    <arg name="baud" default="57600"/>

    <node pkg="multi_uav_se_mission" name="worker_manual_node" type="worker_manual_node" output="screen">
        <!-- parameters -->
        <param name="uavId" value="$(arg uavId)" />
        <param name="altitude" value="$(arg altitude)" />
        <param name="taskTypes" value="$(arg taskTypes)" />
        <param name="taskAssignAtSeconds" value="$(arg taskAssignAtSeconds)" />
        <param name="serialPort" value="$(arg serialPort)" />
        <param name="baud" value="$(arg baud)" />
        <remap from="statistics" to="/uav$(arg uavId)/statistics/data" />
        <remap from="statisticsLog" to="/uav$(arg uavId)/statistics/log" />
        <remap from="statisticsMessages" to="/uav$(arg uavId)/statistics/messages" />
        <remap from="statisticsTasks" to="/uav$(arg uavId)/statistics/tasks" />
    </node>

</launch>
